ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -929,6 +948,22 @@ private void validateExplain(final PreparedStatement<Explain> statement) {
       explain(statement, scopedPropertyOverrides, ksqlConfig, executionSandbox);
     }
 
+    /**
+     * @deprecated `RUN SCRIPT` is deprecated since 5.2 and will be removed in the next major rel.
```

Comment:
nit: what's the point of annotating with @deprecated? this isn't a public api and we're suppressing all the warnings.

Reply:
It makes it easy to search for what should be removed i the next major release.